themes/default/content/blog/launch-redshift-into-aws-vpc-with-pulumi/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Launch an Amazon Redshift cluster into a VPC with Pulumi
+date: 2022-11-07
+meta_desc: In this post, you'll learn how to deploy an Amazon Redshift cluster into an AWS VPC and manage it with Pulumi.
+meta_image: meta.png
+tags:
+    - aws
+    - vpc
+    - redshift
+    - databases
+---
+
+Infrastructure as code (IaC) enables us to manage cloud resources using a declarative approach. We keep declarations of the cloud environment in the text file of the specific format. So, instead of provisioning cloud resources using the Amazon Web Console (AWS), we can set up our cloud environment using a command-line interface and a declaration. This allows us to keep declarations in the source control system and deploy cloud resources as we would with our application.
+
+AWS provides CloudFormation as an IaC tool. However, using CloudFormation isn’t always straightforward. We have a visual editor with the AWS console, enabling us to draw and configure cloud resources to deploy. This approach is very often difficult to learn and use.
+
+To solve this problem and accelerate IaC adoption, Pulumi offers the cross-platform IaC approach, which allows us to choose among the most popular scripting or programming languages, including TypeScript, Python, Go, C#, and Java. With Pulumi, we declare and configure our cloud resources as we would create programming objects in your application code. We can use IaC much faster without learning complicated tools like CloudFormation.
+
+This article shows you how to use Pulumi to deploy an Amazon Redshift cluster in a virtual private cloud in AWS. It also demonstrates how to set up an Amazon S3 bucket and configure the AWS identity and access management (IAM) role so the Amazon Redshift cluster can access objects in the S3 bucket.
+
+Here’s a list of things you need to follow this article:
+
+* [Pulumi](/docs/get-started/aws/) configured for AWS
+* [Node.js](https://nodejs.org/en/download/)
+* Code editor like [Visual Studio Code](https://code.visualstudio.com/)
+
+You'll find the full code for this  [here](https://github.com/dawidborycki/pulumi-redshift).
```

Comment:
should we actually just move this code to pulumi examples?

Reply:
Yep, I'm doing that now.

Reply:
Here's the PR that adds this example (with revisions to use config & secrets and add a template): https://github.com/pulumi/examples/pull/1301